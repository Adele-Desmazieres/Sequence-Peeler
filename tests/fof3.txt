../Tests/t4.fasta
../Tests/Toto/t4.fasta