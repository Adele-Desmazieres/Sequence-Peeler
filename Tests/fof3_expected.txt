t4.fasta
t4_1.fasta